x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlFiltersIT.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ml.integration;
+
+import org.elasticsearch.xpack.core.ml.action.GetFiltersAction;
+import org.elasticsearch.xpack.core.ml.job.config.MlFilter;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class MlFiltersIT extends MlNativeIntegTestCase {
+
+    @Override
+    protected void cleanUpResources() {
+        // nothing to clean
+    }
+
+    public void testGetFilters_ShouldReturnUpTo100ByDefault() {
```

Comment:
#nit 
This seems like a very expensive test. Spinning up 3 nodes and native processes to make filter API calls. 

I suppose it is thorough. Might could move to our single node internal cluster tests at some point.

Reply:
Good point. I fixed it.